stake-pool/program/src/state.rs

```diff
@@ -115,3 +117,66 @@ impl State {
         }
     }
 }
+
+#[repr(C)]
+#[derive(Clone, Copy, Debug, Default, PartialEq)]
+pub struct ValidatorStakeAccountsStore {
+    pub validator_stake_accounts: vec![Pubkey],
+    pub balance: u64,
+    pub balance_update_epoch: u64,
+}
+impl ValidatorStakeAccountsStore {
+}
+
+#[repr(C)]
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub enum StoreState {
+    Unallocated,
+    Init(ValidatorStakeAccountsStore),
+}
+
+impl StoreState {
+    pub const LEN: usize = size_of::<u8>() + size_of::<ValidatorStakeAccountsStore>();
```

Comment:
We need to multiply the length of each record by the constant number of records.